src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -8,6 +8,7 @@
 using System.Reflection.Metadata;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeGen;
+using Microsoft.CodeAnalysis.Debugging;
```

Comment:
Yes. The StateMachineHoistedLocalScope is in this namespace, the other equivalent type that used to be in Cci namespace is deleted.